0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -133,9 +133,8 @@ Retrieve some metadata about the ledger. Plugin must be connected, otherwise the
 {
   "prefix": "us.fed.some-bank.",
   "precision": 10,
```

Comment:
after discussing, @sharafian agreed that the rename to `currencyScale` is defendable.